Library/Formula/shibboleth-sp.rb

```diff
@@ -0,0 +1,106 @@
+class ShibbolethSp < Formula
+  desc "Shibboleth 2 Service Provider daemon"
+  homepage "https://wiki.shibboleth.net/confluence/display/SHIB2"
+  url "http://shibboleth.net/downloads/service-provider/latest/shibboleth-sp-2.5.5.tar.gz"
+  sha256 "30da36e0bba2ce4606a9effc37c05cd110dafdd6d3141468c4aa0f57ce4d96ce"
+
+  option "with-homebrew-httpd22", "Use Homebrew Apache httpd 2.2"
+  option "with-homebrew-httpd24", "Use Homebrew Apache httpd 2.4"
+
+  depends_on "curl" => "with-openssl"
+  depends_on "opensaml"
+  depends_on "xml-tooling-c" => "with-openssl"
+  depends_on "xerces-c"
+  depends_on "xml-security-c"
+  depends_on "log4shib"
+  depends_on "boost"
+  depends_on "homebrew/apache/httpd22" if build.with? "homebrew-httpd22"
+  depends_on "homebrew/apache/httpd24" if build.with? "homebrew-httpd24"
+
+  skip_clean "var/run/shibboleth"
+  skip_clean "var/cache/shibboleth"
+
+  def apache_configdir
+    if build.with? "homebrew-httpd22"
+      "#{etc}/apache2/2.2"
+    elsif build.with? "homebrew-httpd24"
+      "#{etc}/apache2/2.4"
+    else
+      "/etc/apache2"
+    end
+  end
+
+  def install
+    ENV.O2
+    args = [
+      "--disable-debug",
+      "--disable-dependency-tracking",
+      "--disable-silent-rules",
+      "--prefix=#{prefix}"
+    ]
+    args << "--with-xmltooling=#{Formula["xml-tooling-c"].opt_prefix}"
+    args << "--with-saml=#{Formula["opensaml"].opt_prefix}"
+    args << "DYLD_LIBRARY_PATH=#{lib}"
+    if build.with? "homebrew-httpd22"
+      args << "--enable-apache-22"
+    elsif build.with? "homebrew-httpd24"
+      args << "--enable-apache-24"
+    end
```

Comment:
You're correct to say it is not strictly a dependency. But without this, the caveat section instructing the user to place reference to the shib library in their httpd.conf file wouldn't exist and I think it would make the whole process even more confusing than it already is by removing that.